.github/workflows/updateprojects.yml

```diff
@@ -0,0 +1,28 @@
+name: Update Projects
```

Comment:
Will we will need any manual management of the project pages? Or does all info come from the landscape now?

Reply:
All the project info comes from landscape and inserted via hugo shortcodes. There is some peripheral copy and stuff, however, that can still be edited the normal way by editing the content file.